deploy/template.yaml

```diff
@@ -265,6 +269,81 @@ Resources:
       FunctionName: !Ref ClaimedIdentityFunction
       Principal: apigateway.amazonaws.com
 
+  ## UserInfo
+  UserInfoFunction:
+    Type: AWS::Serverless::Function
+    DependsOn: UserInfoFunctionLogGroup
+    Properties:
+      FunctionName: !Sub "User-Info-${AWS::StackName}"
+      Handler: UserInfoHandler.lambdaHandler
+      CodeUri: ../src/
+      Environment:
+        Variables:
+          POWERTOOLS_SERVICE_NAME: UserInfoHandler
+          SESSION_TABLE:
+            Fn::ImportValue: !Sub "${L2DynamoStackName}-session-table-name"
+          KMS_KEY_ARN:
+            Fn::ImportValue: !Sub "${L2KMSStackName}-vc-signing-key"
+      Policies:
+        - AWSLambdaBasicExecutionRole
+        - AWSXrayWriteOnlyAccess
+        - DynamoDBWritePolicy:
+            TableName:
+              Fn::ImportValue:
+                !Sub "${L2DynamoStackName}-session-table-name"
+        - DynamoDBReadPolicy:
+            TableName:
+              Fn::ImportValue:
+                !Sub "${L2DynamoStackName}-session-table-name"
+        - KMSDecryptPolicy:
+            KeyId:
+              Fn::ImportValue:
+                !Sub "${L2DynamoStackName}-session-table-key-id"
+        - Statement:
+            - Sid: KMSPolicy
+              Effect: Allow
+              Action:
+                - kms:*
```

Comment:
Howcomes we're doing a wildcard here?

Reply:
Yeah that needs to be commented. I had set it to only the specific key (commented in line 308) but as part of my kms issue troubleshooting i ended up using wildcard and forgot to revert it back. I will revert this and test it again.

Reply:
Changed to specific key arn value and tested it.

Reply:
This is still allowing a wildcard for actions. Can we refine this to only the permissions needed?